src/main/Extensions/Base64Conversion/Base64Conversion.ts

```diff
@@ -20,6 +26,44 @@ export class Base64Conversion implements Extension {
         private readonly translator: Translator,
     ) {}
 
+    public getInstantSearchResultItems(searchTerm: string): InstantSearchResultItems {
+        const results: Array<InvocationResult> = [];
```

Comment:
Just being nitpicky here: please try to stay consistent with the rest of the code base. Use `T[]` over `Array<T>` when possible:

```suggestion
        const results: InvocationResult[] = [];
```